### PR TITLE
#483 Added Now button to bring timestamp to current date and time.

### PR DESCRIPTION
**Pull request by MattWood21**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [x] UI change (please include screenshot!)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Internationalization and localization
- [ ] Other (please describe):

## What is the current behavior?

Introduces request from #483 which asks to have a Now button. The button could also be labeled "Reset", but the term "Now" is more explicit about what the button intends to do (set all times to the current date and time).

Issue Number: #483 

## What is the new behavior?

- Adds a "Now" button in the Timestamp converter
- Clicking the "Now" button sets time to currnet date and time and resets the display window times
- Behaves similar to how the page is initiated
- Added default transactions for "Now" button

## Other information

image

## Quality check

Before creating this PR, have you:

- [x] Followed the code style guideline as described in [CONTRIBUTING.md](https://github.com/veler/DevToys/blob/main/CONTRIBUTING.md)
- [x] Verified that the change work in Release build configuration
- [x] Checked all unit tests pass

### CODE REVIEW

File: src/dev/impl/DevToys/Strings/fr-FR/Common.resw [hunk 1]
```diff
@@ -168,6 +168,9 @@
   <data name="Megabytes" xml:space="preserve">
     <value>MO</value>
   </data>
+  <data name="Now" xml:space="preserve">
+    <value>À présent</value>
```
Comment (veler) [inline src/dev/impl/DevToys/Strings/fr-FR/Common.resw:172]: I'd suggest to keep the word in English for language that you may not be fluent with, and let people who are comfortable with the language translating it. Here is an example: while `À présent` is correct literal translation of `Now`, in this context, it's incorrect and is confusing. The correct one would be `Maintenant`.

**Review by veler — CHANGES_REQUESTED** [review]
Hello,
Thanks for this contribution. It looks good overall. Can you please change the text in the .resw that have been translated through a tool like Google Translate to keep it in English (`Now`) ? We can then let translators, who are better than Google Translate, translate the language correctly.

Thank you :)

File: src/dev/impl/DevToys/Strings/fr-FR/Common.resw (hunk 1 above)
Comment (MattWood21) [reply src/dev/impl/DevToys/Strings/fr-FR/Common.resw]: Fixed in [cbf2329](https://github.com/veler/DevToys/pull/515/commits/cbf2329c2a35103e35fc16bb30ba6228c47445ad).

I wasn't sure if English or Google Translate was preferred in contributions. Now I know. 👍

**Review by veler — APPROVED** [review]
it looks great now! :D Thank you very much for this !

**Pull request merged**